source/events/events.rst

```diff
@@ -0,0 +1,295 @@
+.. include:: ../references.txt
+
+.. iact_events:
+
+``EVENTS`` extension
+====================
+
+The ``EVENTS`` extension is a binary FITS table that contains an event list.
+Each row of the table provides information that characterises one event.
+The mandatory and optional columns of the table are listed below. In
+addition, a list of header keywords providing metadata is specified.
+Also here there are mandatory and optional keywords.
+The suggested extension name of the binary table is ``EVENTS``.
+
+
+Mandatory columns
+-----------------
+
+* ``EVENT_ID`` tform: ``1J``
+    * Event identification number at the DL3 level
+      (lower data levels could be different, see note below).
+* ``TIME`` tform: ``1D``, unit: s
+    * Time stamp of event in instrument specific MJD time reference. See the
+      header keywords ``MJDREFI`` and ``MJDREFF`` for the zero point of
+      the reference time.
+      See also the `OGIP event list`_ standard.
+* ``RA`` tform: ``1E``, unit: deg
```

Comment:
To specify the column type (using `RA` as an example for my general point here), I would prefer we use:
```
type: float
```
instead of
```
tform: ``1E``
```

The FITS code ``1E`` is very cryptic ... I have to go look it up every time ... is `E` 32 or 64 bits?

Also -- I don't think we should force 32 or 64 bit here in the spec, it's OK if both are valid.
Personally I prefer 32 bit for RA to save space.

There are a few fields where the number of bits matters, e.g. TIME and there we can write ``int64`` which everyone understands immediately without having to look up FITS number codes.

OK?